salesloft/metadata/index.json

```diff
@@ -216,9 +221,49 @@
       "url": "https://developers.salesloft.com/docs/api/call-sentiments-index"
     },
     {
-      "name": "create-conversations-call",
-      "displayName": "Create Conversations Call",
-      "url": "https://developers.salesloft.com/docs/api/create-conversations-call"
+      "name": "conversations-calls-create",
+      "displayName": "Create a new call",
+      "url": "https://developers.salesloft.com/docs/api/conversations-calls-create"
+    },
+    {
+      "name": "conversations-find-all",
+      "displayName": "Fetch conversations",
+      "url": "https://developers.salesloft.com/docs/api/conversations-find-all"
+    },
+    {
+      "name": "conversations-find-one",
```

Comment:
There are more non Read operations that were added since last execution

Reply:
As far as I can tell, this file doesn't get used anywhere, is that right? It's fine that it's here in the repo still, just want to make sure I understand.

Reply:
Index is intermidiate file. It is not used directly. Scrapping is done in 2 phases collecting the index. Usually website will have endpoints in sections.

And then schemas.json would reflect the model scrapped for each index entry. So it is trully an index, like in a book or database.